### PR TITLE
Make the package work with just one pin.

### PR DESCRIPTION
**Pull request by floitsch**
Also improve the way the threshold is computed.

### CODE REVIEW

File: src/dht11.toit [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+  constructor pin/gpio.Pin:
+    pin_ = pin
+    pin.config --input --output --open_drain
+    pin.set 1
+
+    sleep --ms=1000 // Allow the sensor to stabilize after power-up.
+
+  /**
+  Reads the temperature and humidity from the DTH11.
```
Comment (erikcorry) [inline src/dht11.toit]: DHT

File: src/dht11.toit [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  hash_code -> int:
+    return (temperature * 10).to_int * 11 + (humidity * 10).to_int * 13
+
+  /** See $super. */
+  stringify -> string:
+    return "T: $(%.2f temperature), H: $(%.2f humidity)"
+
+class Dht11:
+  static DHT_PULSES_   ::= 41   // Bit pulses produced by DHT sensor. Initialization + 40 bits data.
+  static DHT_MAXCOUNT_ ::= 1000 // Timeout while waiting for edges.
```
Comment (erikcorry) [inline src/dht11.toit]: MAX_COUNT ?

**Review by erikcorry — APPROVED** [review]

File: src/dht11.toit (hunk 2 above)
Comment (floitsch) [reply src/dht11.toit]: done.

File: src/dht11.toit (hunk 1 above)
Comment (floitsch) [reply src/dht11.toit]: done.

**Pull request merged**